index.rst

```diff
@@ -629,7 +629,67 @@ When you connect to the computer running a CSC and edit the configuration direct
   version: heads/tickets/DM-12345-0-g79e2257-dirty
 
 When this happens, it prevents us from precisely identifying what configuration was used.
-In this case, the preferred solution is to use :ref:`section-on-the-fly-config` to ensure traceability is not lost, at the expense of a couple extra minutes.
+In this case, the preferred solution is to follow the `On-the-Fly Configuration Changes`_ process to ensure traceability is not lost, at the expense of a couple extra minutes.
+
+.. _section-configuration-interaction-traceability:
+
+Finding and Using a Previously Used Configuration
+=================================================
+
+In the future, one may want to verify which configuration was being used for a given observation and possibly load the exact same configuration.
+Because we often use generic filenames (e.g. `simple_algorithm.yaml`), and file contents can change with time, creating a robust version controlled system must go beyond simply changing filenames.
+For this reason, additional metadata is associated with each configuration, notably the ``url`` and ``version`` parameters in both the ``configurationsAvailable`` and ``configurationApplied`` events.
+These parameters are key to ensuring that each configuration is unique, and is traceable to their filename and contents.
+
+The ``url`` parameter simply contains a URL indicating how the CSC connects to its settings (meaning a link to the repository).
```

Comment:
I feel this is too vague. Is this a link to the CSC's own repo, or the config repo? Please say whichever it is, rather than "indicating how...(meaning...)".

Also please delete "simply".